src/main/webapp/resources/sass/pages/project-settings-basic.scss

```diff
@@ -0,0 +1,3 @@
+.auto-analysis-status {
```

Comment:
I really try to avoid writing specific css classes for this.  If this is truly a one off for a single css entry then just use an inline style. You could look in app.scss which has classes like `spaced-*`.  There is currently a `spaced-bottom` you could create a reusable class called `spaced-bottom__sm` similar to the other ones there.

Reply:
Fixed by @joshsadam  in 41ab5c1